### DISCUSSION TITLE
status unknown for quota

### DISCUSSION THREAD

**Discussion by vinisman** [okd-project/okd#1122] [General]
Hi all!
After updating okd cluster to the last actual version 4.9.0-0.okd-2022-02-12-140851 I catch an error when i whant to create deploymentConfig:
```
Error creating deployer pod: pods "uikit-1-deploy" is forbidden: status unknown for quota: example, resources: limits.cpu,limits.memory,pods,requests.cpu,requests.memory
```

```
$ oc describe quota example
Name:            example
Namespace:       gpbu20000
Resource         Used  Hard
--------         ----  ----
limits.cpu       0     2
limits.memory    0     2Gi
pods             0     4
requests.cpu     0     1
requests.memory  0     1Gi

```

Please help

**Comment by vrutkovs** [comment]
Looks similar to https://bugzilla.redhat.com/show_bug.cgi?id=1905586?

**Reply by vinisman** [reply]
Hi @vrutkovs. May be these are similar errors but we have another scenario to get it.
1) Create a new project without quota
2) Then create a resourceQuota object
3) Create deploymentConfig

In AdminConsole i have such picture
![image](https://user-images.githubusercontent.com/737526/156057116-1533d2d7-6806-4fbf-a39d-b9e6887c37e1.png)
And the error I mentioned above in events for deploymentConfig and nothing else.

**Reply by linearlyIndependent** [reply]
Hi @vrutkovs, sorry for bumping this old discussion. We encountered the same issue in one of our projects on OKD 4.10, but with different steps. Based on what I observed I don't think it is [https://bugzilla.redhat.com/show_bug.cgi?id=1905586](https://bugzilla.redhat.com/show_bug.cgi?id=1905586). 

 Should I put the details here or open a new discussion?
